Place explicit dependency on clojure.tools.cli

Hi!

I use (and have to use) `[org.clojure/tools.namespace "0.3.0-alpha4"]` and `[org.clojure/tools.reader "1.1.0"]`.

I reckon they depended on clojure.tools.cli at some point, which is why you don't depend on it in your project.clj. So I get: 

```
Caused by: java.io.FileNotFoundException: Could not locate clojure/tools/cli__init.class or clojure/tools/cli.clj on classpath.
clojure.lang.Compiler$CompilerException: java.io.FileNotFoundException: Could not locate clojure/tools/cli__init.class or clojure/tools/cli.clj on classpath., compiling:(leiningen/bikeshed.clj:1:1)
```

Of course, I could fix by depending on `[org.clojure/tools.cli "0.3.5"]` myself.

Just a heads up 🙌

Cheers - Victor

--- a/project.clj
+++ b/project.clj
@@ -5,4 +5,5 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :eval-in :leiningen
-  :dependencies [[org.clojure/tools.namespace "0.2.6"]])
+  :dependencies [[org.clojure/tools.cli "0.3.5"]
+                 [org.clojure/tools.namespace "0.2.6"]])

